.github/workflows/shared-manual-deploy-java.yml

```diff
@@ -40,24 +37,24 @@ jobs:
         uses: cartoncloud/actions/deploy-java@v3
```

Comment:
TODO: These references aren't going to work as we have required changes as part of this PR

Reply:
@aarontwf  need more information, I don't follow

Reply:
unless you're saying that https://github.com/cartoncloud/devops-scripts/pull/1 is a dependency

Reply:
If we wanted to migrate the Java repos gradually, the actions in the repo would need to point to this branch `@tech/CC-26846`. The problem is that inside this PR we have also changed these shared actions but this PR points to the `v3` branch for those so when using them, it will reference the outdated ones. We could change them to `tech/CC-26846` but they'd need to be changed back before merging.

Next step is to figure out how we want to roll out the change across the Java repos, which will determine if we need to worry about any of the things I mentioned above:
- All at once
- One (i.e. service-integrations), then the rest all at once
- One at a time

Reply:
My plan is to run https://github.com/cartoncloud/devops-scripts/pull/1 script
and shortly afterwards, merge this to v3
I think each of the java repos will then work - this needs testing (from v3 - roll forward with any fixes), that's the plan
And later, I'll remove the environment secrets